app/views/coronavirus_form/medical_conditions.html.erb

```diff
@@ -0,0 +1,30 @@
+<% content_for :title do %><%= t('coronavirus_form.medical_conditions.title') %><% end %>
+    <% content_for :meta_tags do %>
+      <meta name="description" content="<%= t('coronavirus_form.medical_conditions.title') %>" />
+    <% end %>
+    
+    <% content_for :back_link do %>
+      <%= render "govuk_publishing_components/components/back_link", { href: previous_path } %>
+    <% end %>
+    
+    <%= form_tag({},
+      "data-module": "track-coronavirus-form-vulnerable-people-medical_conditions",
+      "data-question-key": "medical_conditions",
+      "id": "medical_conditions"
+    ) do %>
+    <%= render "govuk_publishing_components/components/radio", {
+      heading: t('coronavirus_form.medical_conditions.title'),
+      is_page_heading: true,
+      name: "medical_conditions",
+      description: t('coronavirus_form.medical_conditions.description').html_safe,
```

Comment:
Sorry, I should've suggested `sanitize()` instead of `.html_safe`:

```suggestion
      description: sanitize(t('coronavirus_form.medical_conditions.description')),
```